admin/README.md

```diff
@@ -1,10 +1,349 @@
+- If an `token` property is present, it will be looked up by token
+
+```js
+accountAdmin.accounts.find('account123')
+accountAdmin.accounts.find({username: 'pat@example.com'})
+accountAdmin.accounts.find({token: 'token123'})
+```
+
+### findAll([options])
+
+```js
+accountAdmin.accounts.findAll()
+```
+
+### update(idOrObject, change[, options])
+
+If `idOrObject` is a string, the account will be looked up by accountId. If it's
+an object:
+
+- If a `username` property is present, it will be looked up by username
+- If an `id` property is present, it will be looked up by accountId
+- If an `token` property is present, it will be looked up by token
+
+`change` can be an object of properties that you want to change, or a function
+that receives the current account as argument and can manipulate it directly
+
+```js
+accountAdmin.accounts.update({token: 'token123'}, {password: 'newsecret'})
+accountAdmin.accounts.update('account123', function (account) {
+  account.invites--
+})
+```
+
+### remove(idOrObject[, change, options])
+
+If `idOrObject` is a string, the account will be looked up by accountId. If it's
+an object:
+
+- If a `username` property is present, it will be looked up by username
+- If an `id` property is present, it will be looked up by accountId
+- If an `token` property is present, it will be looked up by token
+
+Before deletion, the account can optionally be updated, e.g. to add a reason
+for the deletion. `change` can be an object of properties that you want to
+change, or a function that receives the current account as argument and can
+manipulate it directly
+
+```js
+accountAdmin.accounts.remove('account123')
+accountAdmin.accounts.remove({
+  username: 'pat@example.com'
+}, {
+  reason: 'support request #123'
+})
+```
+
+## requests
+
+
+Example request Object:
+
+```js
+{
+  id: 'request123',
+  type: 'passwordreset',
+  contact: 'pat@example.com'
+}
+```
+
+### add(object[, options])
+
+```js
+accountAdmin.requests.add({
+  type: 'passwordreset',
+  contact: 'pat@example.com'
+})
+```
+
+### find(idOrObject[, options])
+
+```js
+accountAdmin.requests.find('token123')
+accountAdmin.requests.find({id: 'token123'})
+```
+
+### findAll([options])
+
+```js
+accountAdmin.requests.findAll()
+```
+
+
+### remove(idOrObject[, options])
+
+```js
+accountAdmin.requests.remove('token123')
+accountAdmin.requests.find({id: 'token123'})
+```
+
+## account(idOrObject)
+
+The `.account` method returns a scoped API for one account
+
+```js
+accountAdmin.account('account123')
+accountAdmin.account({id: 'account123'})
+accountAdmin.account({username: 'pat@example.com'})
+```
+
+### account.profile
+
+#### profile.find([options])
+
+```js
+account.profile.find()
+```
+
+#### profile.update(change[, options])
+
+```js
+account.profile.update({
+  fullname: 'Dr. Pat Hook'
+})
+```
+
+### account.tokens
+
+#### tokens.add(object[, options])
+
+```js
+account.tokens.add({
+  type: 'passwordreset',
```

Comment:
I think it would be nicer to have an explicit function like `accountAdmin.forgotPassword('email@address.com')` It would be a little less confusing than `account.tokens.add`. And it can do the account.tokens.add underneath.

If I was building an app with Hoodie. The first thing I would do would be to create a wrapper function around `account.tokens.add` for password resets otherwise I would immediately forget how to do that. So I would prefer we just do that already.